src/passkey_child/passkey_child_assert.c

```diff
@@ -453,19 +413,17 @@ verify_assert(struct pk_data_t *pk_data, fido_assert_t *assert)
 
 void
 print_assert_data(const char *key_handle, const char *crypto_challenge,
-                  const char *auth_data, const char *signature,
-                  const unsigned char *user_id)
+                  const char *auth_data, const char *signature)
 {
     json_t *passkey = NULL;
     char* string = NULL;
-    const char *id = (user_id == NULL) ? "" : (const char *)user_id;
 
     passkey = json_pack("{s:s*, s:s*, s:s*, s:s*, s:s*}",
                         "credential_id", key_handle,
                         "cryptographic_challenge", crypto_challenge,
                         "authenticator_data", auth_data,
                         "assertion_signature", signature,
-                        "user_id", id);
+                        "user_id", "");
```

Comment:
Hi,

the patch is working fine for me. I would just like to suggest to add a comment here about `user_id` so that when someone reads the code and has the same question as Justin the comment will answer it.

bye,
Sumit

Reply:
Done